File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/startup/SelectUserActivity.java
package org.jellyfin.androidtv.startup;

import android.app.Activity;
import android.os.Bundle;

import org.jellyfin.androidtv.R;


public class SelectUserActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_select_user);
    }
}

File: app/src/main/java/org/jellyfin/androidtv/ui/startup/SelectUserActivity.kt
package org.jellyfin.androidtv.ui.startup

import android.os.Bundle
import androidx.fragment.app.FragmentActivity

class SelectUserActivity : FragmentActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)

		supportFragmentManager
			.beginTransaction()
			.replace(android.R.id.content, SelectUserFragment())
			.commit()
	}
}
